### PR TITLE
try to settle the "triangle" issue

### PR DESCRIPTION
**Pull request by YifangSun**
closed

### CODE REVIEW

File: pyxel/core/src/pyxelcore/graphics.cc [hunk 1]
```diff
@@ -254,6 +254,66 @@ void Graphics::DrawCircleBorder(int32_t x,
...
+  float alpha12 = (y2==y1) ? 0 : static_cast<float>(x2 - x1) / (y2 - y1);
+  float alpha13 = (y3==y1) ? 0 : static_cast<float>(x3 - x1) / (y3 - y1);
+  float alpha23 = (y3==y2) ? 0 : static_cast<float>(x3 - x2) / (y3 - y2);
+  int32_t x_intersection = x1 + alpha13 * (y2 - y1)  + 0.5f;
+  int32_t y_slider = y1;
+  for(; y_slider <= y2; y_slider++){
+      int32_t x_slider, x_end;
+
+      if(x_intersection < x2){
+        x_slider = x_intersection + alpha13 * (y_slider - y2) + 0.5f;
```
Comment (YifangSun) [inline pyxel/core/src/pyxelcore/graphics.cc]: compute `x_slider` and `x_end` using `x_intersection`, could avoid the case that y1==y2 and y2==y3

**Comment by kitao** [comment]
It looks like you closed the pull request, but does this mean I shouldn't merge this code?

**Comment by YifangSun** [comment]
For some network reasons, I can't upload my binary files like"libpyxelcore.dll" which works well in my python. I think it's an incomplete PR, so I close it. 
If you don't mind, I can reopen it.(only code, no binary, you should make it) 
Shall I?

**Comment by kitao** [comment]
No I don't. I'll build the binaries.
Please send PR again.

**Comment by kitao** [comment]
Thank you!

**Pull request merged**